### PR TITLE
HGLRC: Add HGLRCF722MINIV2 board target

### PR DESCRIPTION
**Pull request by Linjieqiang**
Link: https://github.com/betaflight/unified-targets/pull/1228

### CODE REVIEW

**Review by asizon — APPROVED** [review]

File: configs/HGLRCF722MINIV2/config.h [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 179
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI2
+#define FLASH_SPI_INSTANCE SPI3
+#define USE_SPI_GYRO
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW0_DEG
+#define PINIO1_BOX 40
+#define PINIO2_BOX 0
```
Comment (haslinghuis) [inline configs/HGLRCF722MINIV2/config.h:112]:
```suggestion
#define PINIO2_BOX 41
```

File: configs/HGLRCF722MINIV2/config.h [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+#define PIONIO2_CONFIG 129
```
Comment (haslinghuis) [inline configs/HGLRCF722MINIV2/config.h:113]:
```suggestion

```

File: configs/HGLRCF722MINIV2/config.h [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+#define ADC3_DMA_OPT 0
+#define ADC_INSTANCE ADC3
+
+#define MAG_I2C_INSTANCE  (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 179
```
Comment (haslinghuis) [inline configs/HGLRCF722MINIV2/config.h:104]:
```suggestion
#define DEFAULT_VOLTAGE_METER_SCALE 110
#define DEFAULT_CURRENT_METER_SCALE 179
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Keep in sync with unified target

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**